src/logic/operate.js

```diff
@@ -0,0 +1,33 @@
+const BigNum = require('big.js');
+
+const operate = (num1, num2, operation) => {
+  let finalRes = new BigNum(0);
+  const a = new BigNum(num1);
+  const b = new BigNum(num2);
+
+  switch (operation) {
+    case '÷':
+      try {
+        finalRes = a / b;
+      } catch (error) {
+        finalRes = 'Er';
+      }
+      break;
+    case 'x':
+      finalRes = a * b;
+      break;
+    case '-':
+      finalRes = a - b;
+      break;
+    case '+':
+      finalRes = a + b;
+      break;
+    default:
+      break;
+  }
+
+  return finalRes.toString();
+};
```

Comment:
For this part of the logic, we need to make use of the logic provided by the [big-js library we imported](https://github.com/favio41/big-js#readme); that means that the operations need to be parsed with the library E.G:  you are going to do an addition, the big-js library way to use it will be `x.plus(y)`; let's use that logic throught this module please, because the point of using the big-js library is that when it comes to a lot of decimals or a really big number JS on it's own is not very trustworthy.